pkg/storage/s3/s3.go

```diff
@@ -111,13 +113,26 @@ func GetConfig(listers *regopclient.Listers) (*S3, error) {
 		}
 	}
 
+	cloudConfig, err := listers.OpenShiftConfigManaged.Get(defaults.KubeCloudConfigName)
```

Comment:
For OpenStack, the "ca-bundle.pem" is grabbed from the openshift-config/cloud-provider-config ConfigMap. I have opted to use what I think is the more appropriate openshift-config-managed/kube-cloud-config ConfigMap. In the long term, we should probably be consistent, however.